lib/themes/dosomething/paraneue_dosomething/js/reportback/Reportback.js

```diff
@@ -308,7 +327,7 @@ const Reportback = {
         'isListItem': true,  // @TODO: probably a better way to address this... :insert shruggy emoji here:
         'reactions': {
           enabled: setting('dsKudosReactions.enabled'),
-          data: this.getReactions(items[i].kudos.data),
+          data: setting('dsKudosReactions.likedByCurrentUser') ? this.getReactions(items[i].kudos, true) : this.getReactions(items[i].kudos.data, false),
         }
```

Comment:
Yeah! Wherever possible, I think it'll be easier to try to make the Rogue endpoint match the old Phoenix endpoint's structure as close as we can.